.github/workflows/gradle.yaml

```diff
@@ -256,23 +256,37 @@ jobs:
       - name: Build
         run: |
           rustup target add aarch64-apple-darwin
+          tag=${GITHUB_REF#refs/*/}
+          if [[ "$tag" == v* ]]; then
+            echo "Overwriting version: $tag"
+            cd gui
+            pnpm version "$tag" --force --allow-same-version --workspaces-update false
+            git add package.json
+            cd ..
+            pnpm version "$tag" --force --allow-same-version --workspaces-update false
+          fi
```

Comment:
```suggestion
```
i have another way to fix this, but it's gonna be in another PR.
Mind removing this? :3